src/main/java/com/csselect/database/mysql/MysqlGameAdapter.java

```diff
@@ -0,0 +1,325 @@
+package com.csselect.database.mysql;
+
+import com.csselect.Injector;
+import com.csselect.database.DatabaseAdapter;
+import com.csselect.database.GameAdapter;
+import com.csselect.game.BinarySelect;
+import com.csselect.game.Feature;
+import com.csselect.game.FeatureSet;
+import com.csselect.game.Gamemode;
+import com.csselect.game.MatrixSelect;
+import com.csselect.game.NumberOfRoundsTermination;
+import com.csselect.game.Round;
+import com.csselect.game.Termination;
+import com.csselect.game.TerminationComposite;
+import com.csselect.game.TimeTermination;
+import com.csselect.user.Organiser;
+import com.csselect.user.Player;
+import com.csselect.utils.FeatureSetUtils;
+
+import java.io.IOException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.StringJoiner;
+
+/**
+ * Mysql-Implementation of the {@link GameAdapter} Interface
+ */
+public class MysqlGameAdapter extends MysqlAdapter implements GameAdapter {
+
+    private static final MysqlDatabaseAdapter DATABASE_ADAPTER
+            = (MysqlDatabaseAdapter) Injector.getInjector().getInstance(DatabaseAdapter.class);
+
+    private String databaseName;
+
+    /**
+     * Creates a new {@link MysqlGameAdapter} with the given id
+     * @param id adapters id
+     */
+    MysqlGameAdapter(int id) {
+        super(id);
+    }
+
+    /**
+     * Creates a new {@link MysqlGameAdapter} with the next available id
+     * @throws SQLException Thrown if an error occurs while communicating with the database
+     */
+    MysqlGameAdapter() throws SQLException {
+        super(DATABASE_ADAPTER.getNextGameID());
+        DATABASE_ADAPTER.executeMysqlUpdate("INSERT INTO games () VALUES ();");
+    }
+
+    @Override
```

Comment:
Is this method overwrite neccissary

doesn't the method of the parent get called by default if there is no implementation